added description parameter

vicinus opened the pull request:
This adds the option to add a short description to a cron job. If you are generally willing to merge, I can add documentation, spec tests etc.

roman-mueller commented:
Absolutely interested!
Can you also add the `$description` to https://github.com/roman-mueller/rmueller-cron/blob/master/manifests/job/multiple.pp (and that template)?

I think comments make especially sense in those files with multiple cronjobs.

vicinus commented:
ok, I have updated the pull request. The parameter description can be added to multiple jobs also. I extended the spec test  'job with custom values' and added documentation. 

As far as I can tell the problem with the check is not connected to my changes. Let me know if I'm wrong or if I can improve anything else.

roman-mueller commented:
:+1: 

You are right, the failing test is not related to your change from what I can tell.
I think it's time to deprecate the rvm 1.8.7 checks.

Thanks for your contribution!
